src/main/webapp/style.css

```diff
@@ -53,6 +55,31 @@ body {
   color: white;
 }
 
+#close-directions {
+  visibility: hidden;
+}
+
+#location-info {
+  color: grey;
+  visibility: hidden;
+}
+
+#close-sidebar {
+  margin-left: 300px;
+}
+
+#open-sidebar {
+  display: none;
+}
+
+#route-info {
+  background-color: white;
+  border: 2px solid grey;
+  display: inline-block;
+  padding: 0 10px 0 10px;
+  visibility: hidden;
+}
+
 .sidenav {
```

Comment:
lines 95-105 is now irrelevant code - feel free to remove them when changing sidenav class to sidebar id @purple-waves

Reply:
Is that something that was irrelevant before?

Reply:
i couldnt highlight it before because it was my code but it was formating for the text in the side bar

Reply:
Removed